pace-util/pace/util/zarr_monitor.py

```diff
@@ -126,6 +128,18 @@ def store(self, state: dict) -> None:
         for name, quantity in sorted(state.items(), key=lambda x: x[0]):
             self._writers[name].append(quantity)  # type: ignore[index]
 
+    def store_grid(self, grid: dict) -> None:
```

Comment:
Can you remove the name reference to "grid" in this file, and instead use a generic term like "constant"? This logic should work for anything that isn't time-varying.

Reply:
Changed to `store_constant`